src/models/health.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const db = require('../lib/db')
+
+function * getDbHealth () {
+  const isConnected = yield db.isConnected()
+  return isConnected ? {
+    status: 'OK'
+  } : {
+    status: 'KO'
```

Comment:
There is another convention in the connector: https://github.com/interledger/five-bells-connector/blob/master/src/common/health.js